### ISSUE TITLE
cannot find macro `jsonapi_model!` in this scope

### ISSUE THREAD

**Issue by JadedEvan** [michiel/jsonapi-rust#32]
I'm looking for a little guidance on what would seem to be a very contrived and easy example. I have an extremely bare-bones application defined in `src/main.rs` as follows:

```rust
#[macro_use] extern crate jsonapi;

use jsonapi::model::*;

#[derive(Debug)]
pub struct Person {
    pub id: i32,
    pub name: String
}

jsonapi_model!(Person; "person");

fn main() {
    let user = Person { id: 1, name: "World".to_string() };
    println!("{:?}", user);
}
```

Upon compilation I receive the following error

    cannot find macro `jsonapi_model!` in this scope

What am I missing here? I feel like I've copied the necessary includes and macro imports (based on what I was seeing in the `tests` directory) yet I cannot get this to compile. I am running `rustc 1.22.1`. Any help would be greatly appreciated.

**Comment by michiel** [comment]
I think you might be using the last published crate, which doesn't have this functionality yet.

Can you try updating your `Cargo.toml` to pull master branch from github?

    [dependencies]
    jsonapi = { git = "https://github.com/michiel/jsonapi-rust", branch = "master" }

**Comment by JadedEvan** [comment]
@michiel yes this completely solves the problem. Thanks for the solution.

Out of curiosity - what's the expectation around publishing a crate with the macro included? Are you anticipating that you'll push out a version for this soon? This solution is easy enough - I just worry that other people may stumble into this same issue without any prior knowledge.

**Comment by michiel** [comment]
@JadedEvan Sorry that took a bit longer, the recently published v0.5.4 and 0.6.0 both contain the macro.
